pkgs/development/python-modules/eth-utils/default.nix

```diff
@@ -35,6 +35,9 @@ buildPythonPackage rec {
     pytestCheckHook
   ] ++ eth-hash.optional-dependencies.pycryptodome;
 
+  # Skip Python tests due to mypy related test failure
```

Comment:
Can we just skip mypy instead? Also we don't even want to check mypy because it is a linter.

Reply:
I can check, the least elegant way could be simply deleting those tests.

Reply:
```

  # Removing a poorly written test case from test suite.
  # TODO work with the upstream
  disabledTestPaths = [
    "tests/functional-utils/test_type_inference.py"
  ];
  ```


Done, this should be elegant enough. To be squashed.